libpod/container_internal_common.go

```diff
@@ -388,7 +388,7 @@ func (c *Container) generateSpec(ctx context.Context) (s *spec.Spec, cleanupFunc
 			Destination: dstPath,
 			Options:     bindOptions,
 		}
-		if c.IsReadOnly() && (dstPath != "/dev/shm" || !c.config.ReadWriteTmpfs) {
+		if c.IsReadOnly() && (dstPath != "/dev/shm" || !c.config.ReadWriteTmpfs) && dstPath != "/catatonit" {
```

Comment:
It would be best to mount catatonit always read only.

Also hard coding liek this does not seem nice, in your lookup code you used filepath.Base(catatonitPath) which means it could open up drift between these two code paths.

Reply:
is this still valid?

Why don't we use `filepath.Base(c.FindInitBinary())` here?

What happens if I do something like `-v /foo:/catatonit`?  Will it be forced to be `ro`?  Anyway it seems redundant since we've already added `ro`

Reply:
I've removed it in latest update.